server.js

```diff
@@ -33,12 +37,13 @@ app.use(passport.initialize());
 app.use(passport.session());
 
 // Serve up static assets
-app.use(express.static("client/build"));
```

Comment:
I think we will need this Dave for when we put site into production enviroment and run 'yarn build'. Can you just comment this out for now?

Reply:
Let's get clarification here. I think this is only in our file because we copied the skeleton from one of our assignments. I can't see why we would use this to serve up static assets